### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by BlockLegend001** [community/community#65898] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How I can do a pull request?

**Comment by Nom4dx** [comment]
Just go to the official documentation: https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github- enterprise/creating-an-issue-or-pull-request-from-github-desktop

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
